ambry-tools/src/main/java/com.github.ambry/store/DumpDataTool.java

```diff
@@ -90,7 +90,7 @@ public static void main(String args[]) throws Exception {
    * Executes the operation with the help of properties passed
    * @throws IOException
    */
-  private void doOperation() throws Exception {
+  public void doOperation() throws Exception {
```

Comment:
So that if anyone outside of ambry wants to leverage our tools. Hence made the class and this method public